DataStructure/Queue.py

```diff
@@ -1,18 +1,31 @@
 # queue works on first in first out principle(fifo)
 
-queue = []
+class Queue:
+    def __init__( self ):
+        self. items = []
+
+    def isEmpty (self):
+        return self. items == []
+
+    def enqueue (self, item):
+        self. items. insert ( 0, item )
+
+    def dequeue (self):
+        return self. items. pop()
+
+    def size (self):
+        return len (self.items)
+    
+q=Queue ()
```

Comment:
space rules are space around operator, but no space between after . and after name of function and ()
q = Queue()
q.enqueue("Deer")